modules/cnf-migrating-from-amqp-to-http-transport.adoc

```diff
@@ -0,0 +1,184 @@
+// Module included in the following assemblies:
+//
+// * networking/using-ptp.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-migrating-from-amqp-to-http-transport_{context}"]
+= Migrating consumer applications to use HTTP transport for PTP and bare-metal events
+
+If you have previously deployed PTP or bare-metal events consumer applications, you need to update the applications to use HTTP message transport.
+
+You need to configure persistent storage for message subscriptions, set the `transportHost` in the PTP or {redfish-operator} configuration, and update existing consumer applications to use the new HTTP `transportHost`.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in as a user with `cluster-admin` privileges.
+
+* You have updated the PTP Operator or {redfish-operator} to version 4.13+ which uses HTTP transport by default.
+
+.Procedure
+
+. Configure persistent storage to persist events subscriptions in the pod managed by the PTP Operator or {redfish-operator}.
+For example, apply the following `PersistentVolumeClaim` (PVC) custom resource (CR) in the cluster:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: cloud-event-proxy-store
+spec:
+  storageClassName: local-storage
+  resources:
+    requests:
+      storage: 10Mi
+  accessModes:
+  - ReadWriteOnce
+----
+
+. Configure the PTP events producer application with a corresponding `persistentVolumeClaim` that matches the PVC that you have configured in step 1. For example:
++
+[source,yaml]
+----
+volumes:
+  - name: pubsubstore
+    persistentVolumeClaim:
+      claimName: cloud-event-proxy-store
+----
+
+. Get the name of the PTP or bare-metal events `transportHost` in the cluster. For example:
+
+.. To get the name of the PTP events `transportHost`, run the following command:
++
+[source,terminal]
+----
+$ oc get ptpoperatorconfig -n openshift-ptp -o jsonpath='{ .items[*].spec.ptpEventConfig.transportHost }'
+----
++
+.Example output
+[source,terminal]
+----
+http://ptp-event-publisher-service-compute-1-example.openshift-ptp.svc.cluster.local:9043
+----
+
+.. To get the name of the bare-metal events `transportHost`, run the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-bare-metal-events get hardwareevent -o jsonpath='{.items[*].spec.transportHost}{"\n"}'
+----
++
+.Example output
+[source,terminal]
+----
+http://hw-event-publisher-service.openshift-bare-metal-events.svc.cluster.local:9043
+----
+
+. Update your events consumer application to use HTTP transport.
+
+.. Set the `transport-type` variable in the cloud events consumer application deployment, for example:
++
+[source,yaml]
+----
+containers:
+  - name: cloud-event-consumer
+    image: <your_cloud_event_consumer_app_image>
+    args:
+      - "--local-api-addr=127.0.0.1:9089"
+      - "--api-path=/api/ocloudNotifications/v1/"
+      - "--api-addr=127.0.0.1:8089"
+      - "--transport-type=HTTP"
```

Comment:
The manifests in cloud-event-proxy/examples/consumer/deployment/ are obsolete. Please use examples from  cloud-event-proxy/examples/manifests/. The arg `"--transport-type=HTTP"` is no longer needed.